server/chat-commands/info.ts

```diff
@@ -2610,13 +2610,18 @@ export const commands: ChatCommands = {
 		}
 
 		this.checkBroadcast(true, '!code');
-
-		const code = Chat.getReadmoreCodeBlock(target);
 		this.runBroadcast(true);
+
+		const isPMOrPersonalRoom = this.room?.settings.isPersonal !== false;
+
 		if (this.broadcasting) {
-			return `/raw <div class="infobox">${code}</div>`;
+			if (isPMOrPersonalRoom) {
+				target = Chat.filter(this, target, user, room, connection, this.pmTarget)!;
```

Comment:
tbh do we still need `connection` if we refactor `Chat.filter` to use `Chat.ErrorMessage`?

Reply:
I think if we move this `if`-block directly under `isPMOrPersonalRoom`, no other changes are necessary.

Reply:
On second thought, I think this is better.

Reply:
> tbh do we still need connection if we refactor Chat.filter to use Chat.ErrorMessage?

Unless we're doing something weird in `config.js`, I don't think we even need `Chat.ErrorMessage`; no chatfilters in public or private code use `connection`.

Reply:
Yeah, that's why I changed tune to just saying `this` (`CommandContext`) is enough; you can pull anything you want out of the context if you actually need it.